### PR TITLE
Content review fcb68 69 70 71

### LINKED ISSUE #428: Missing persistent id in some recipes
Hi, some recipes in the introduction section. e.g. _Knowledge graphs and FAIR_ and _FAIR and the notion of metadata_, don't have a persistent FCB0XX ID.  Is it possible to generate IDs for such recipes? Thanks!

### PR DESCRIPTION
**Pull request by FuqiX**
close #428 by reviewing FCB68,69. FCB70 and 71 have been reviewed previously.

### CODE REVIEW

**Comment by FuqiX** [comment]
I think it was duplicated. Please correct me if I am wrong

There is one OMOP CDMv5 entry within the EBM NLP box. (original L95)
image
and another box talking about the same dataset?  (original L158)
image

I merged information in these two boxes. and named it "Electronics Medical Notes: SynPUF 1000 person dataset"

File: content/recipes/introduction/training-datasets.md [hunk 1]
```diff
@@ -67,76 +63,82 @@ and mortality (both in and out of hospital).
...
-- **Availability**: https://github.com/bepnye/EBM-NLP
 - **Purpose**: Corpus for training and testing Natural Language Processing algorithms
-- **License**: http://www.apache.org/licenses/LICENSE-2.0
-
+- **Availability**: https://github.com/bepnye/EBM-NLP
+- **Format**: *ad-hoc*, UTF-8 text file with tab-delimited values
+- **License**: [APACHE license, Version 2.0](http://www.apache.org/licenses/LICENSE-2.0)
+````
 
+### Electronics Medical Notes: SynPUF 1000 person dataset
```
Comment (proccaserra) [inline content/recipes/introduction/training-datasets.md:93]:
```suggestion
### Electronics Medical Notes: SynPUF 1000 person dataset & OMOP-CDM v5 standard
```

**Comment by proccaserra** [comment]
@FuqiX good catch. I had to renamed the section title to make it more obvious the link to OMOP CDM.
should be good to go.

**Comment by FuqiX** [comment]
now fixed. Please let me know if further changes are required.

**Review by proccaserra — APPROVED** [review]
looks good to me. thx @FuqiX

**Pull request merged**